### PR TITLE
Added Tiberian Sun map Pit Or Plateau

### PR DESCRIPTION
**Pull request by Mailaender**
This map is open and does not contain (vital) bridges. The layout is balanced and conservative. See http://ra2maps.zzattack.org/Tiberian%20Sun/Official%20Multiplayer/getmap.php?map=Pit+or+Plateau

### CODE REVIEW

File: mods/ts/maps/pitplats/map.yaml [hunk 1]
```diff
@@ -0,0 +1,837 @@
+MapFormat: 7
+
+RequiresMod: ts
+
+Title: Pit Or Plateau
+
+Description: Describe your map here
```
Comment (Unit158) [inline mods/ts/maps/pitplats/map.yaml]: !

Comment (Mailaender) [inline mods/ts/maps/pitplats/map.yaml]: Added some descriptions.

**Comment by GraionDilach** [comment]
One of my favourite maps from TS. (y) ofc.

**Comment by pchote** [comment]
If this is the same as the map we played online, then the player setup for Neutral isn't correct - their structures show up in the "All Players" observer mode.

**Comment by pchote** [comment]
This also needs a larger cordon on the north edge of the map to prevent the glitches at the top of the minimap

screen shot 2015-08-04 at 22 36 52

**Comment by pchote** [comment]
The southern spawn is right on the map border. IIRC from the other maps you need to add 4 to the map height to fix the bottom edge.  You'll be able to compare the painted terrain with the cordon in the editor to find the right value.

**Comment by abcdefg30** [comment]
> If this is the same as the map we played online, then the player setup for Neutral isn't correct - their structures show up in the "All Players" observer mode.

Looks like a regression to me, as this happens in all maps with actors owned by neutral.
(See `Hot Springs`, `Tread Lightly` or `Center of Attention redux 2` [oil derricks there].)

**Comment by Mailaender** [comment]
Fixed in https://github.com/OpenRA/OpenRA/pull/8964.

**Comment by Mailaender** [comment]
If I change map size or bounds slightly the game crashes so I am giving up here. It was uploaded at http://resource.openra.net/maps/3953 for playtesting.

**Comment by pchote** [comment]
Unfortunately my WIP importer still needs a bunch of work before it will be able to import maps that don't need manual tweaking.

**Pull request closed without merging**